docs/int/openapi.yaml

```diff
@@ -1639,7 +1872,21 @@ components:
       schema:
         type: string
         format: uuid
-
+    IdPath:
+      name: id
+      in: path
+      description: id of a resource
+      required: true
+      schema:
+        type: string
+        format: id
+    NameReferenceType:
```

Comment:
Rename to 'NamedReferenceTypeQueryRequired' to match other required query parameter naming. Also, should be added in alphabetical order within the components/parameters section.
- [ ] rename schema